### PR TITLE
Fix "Apply now" button on join-us page

### LINKED ISSUE #756: Add stylings to the apply button on join us page
**Describe the bug**
The button doesn't have a raising animation like all other button. Also when hovering, the color of text turns to blue.

**To Reproduce**
1. Go to 'https://sefglobal.org/join-us.html'
2. Scroll down
3. See error

**Expected behavior**
A raising animation for the button
When hovered, the colour shouldn't change

**Screenshots**
IMG_20200825_232837

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Ubuntu 19.04
 - Browser [e.g. chrome, safari] Google Chrome

**Additional context**
You can refer other pages to fix this

### PR DESCRIPTION
**Pull request by AlexARNcode**
## Purpose
The purpose of this PR is to fix #756

## Goals
We need to make both buttons the same styling, which is not the case right now.

## Approach
I'm using this as a guideline : https://stackoverflow.com/questions/2906582/how-to-create-an-html-button-that-acts-like-a-link
I'm nesting a "div" with the button css style, inside a "a".

### Preview Link

https://pr-760-sef-site.surge.sh/join-us.html

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Learning
I tried to use good practices for buttons, see : https://stackoverflow.com/questions/2906582/how-to-create-an-html-button-that-acts-like-a-link

### CODE REVIEW

File: src/main/webapp/join-us.html
Comment (Piumal1999) [inline src/main/webapp/join-us.html]: It's better if we can indent the code like this  -->
```
<a href="apply.html">
     <button class="btn btn-purple text-uppercase mt-3">
          Apply now!
     </button>
</a>
```

Comment (Piumal1999) [reply src/main/webapp/join-us.html:206]: Also we can remove the href attribute in the button

Comment (Piumal1999) [reply src/main/webapp/join-us.html]: Also we can remove the `href` attribute in the button tag

**Review by Piumal1999 — CHANGES_REQUESTED** [review]
Well done @AlexARNcode !!! We can merge this after some small changes. I have added some comments. If you have any questions, feel free to ask via gitter or sef hive(https://sef.discourse.group)

And could you please copy and fill the pull request template so it will be easy for us to review. 
Link: https://github.com/sef-global/sef-site/blob/master/pull_request_template.md

File: src/main/webapp/join-us.html
Comment (AlexARNcode) [reply src/main/webapp/join-us.html]: No problem, I'll do that :) !
Actually, it looks like a button shouldn't be nested in a "a href". If you try to validate the HTML code in https://validator.w3.org for example, it will tell you that.

Comment (AlexARNcode) [reply src/main/webapp/join-us.html]: > Also we can remove the `href` attribute in the button tag

Yep, already did for this button, but we should do it for the other indeed, and/or do what I told you in my previous comment ?

Comment (Piumal1999) [reply src/main/webapp/join-us.html]: Oh right. Then let's try adding `!important` rule for the color of "btn-purple" class. So we can keep the join-us.html file same as before

Comment (AlexARNcode) [reply src/main/webapp/join-us.html]: Hi mate,
What I meant is this : https://stackoverflow.com/questions/2906582/how-to-create-an-html-button-that-acts-like-a-link

Either the HTML way (with form) or CSS way (create a "a" styled as a button with css). Tell me what you preefer and I'll do it ;)

Comment (Gravewalker666) [reply src/main/webapp/join-us.html]: Hello @AlexARNcode ,
Let's keep only the button tag and place the href as a prop of the button instead of wrapping the button in an `<a>` or adding CSS to the `<a>` to look like button.

**Review by Gravewalker666 — CHANGES_REQUESTED** [review]
@AlexARNcode Thanks for keeping the conversation alive mate! 
Please fill out the generated PR template form so that we all can have a better understanding of what this PR does and how you did it. So that even after merging the PR we can get an idea of what happened to the code and how and why the changes were made.

**Comment by AlexARNcode** [comment]
I did everything, but now it seems I'm having trouble to pull request my changes, oops...

**Review by AlexARNcode — COMMENTED** [review]
Forget this commit, I'm not a github ninja yet, as you can see.. Sorry for that.

**Comment by Gravewalker666** [comment]
No worries at all, take your time my friend. No need to apologize. We've all been there once.
Try to find solutions for the issues that prompt, try googling and learn to find solutions and learn how to apply 'em, not just for this PR, that skill would help you throughout your carrier.

**Comment by AlexARNcode** [comment]
> No worries at all, take your time my friend. No need to apologize. We've all been there once.
> Try to find solutions for the issues that prompt, try googling and learn to find solutions and learn how to apply 'em, not just for this PR, that skill would help you throughout your carrier.

Thank you so much, that's just a so nice comment man. I actually did 3 commits today, just the first one is the good one.

**Comment by AlexARNcode** [comment]
Here is my (good) commit : https://github.com/sef-global/sef-site/pull/760/commits/fe1a62cb9655ccdbd13b81a013ff0c8e25707854 I'm waiting for it to be reviewed (if I understand correctly the workflow).

**Review by Gravewalker666 — APPROVED** [review]
@AlexARNcode Good job dude, well done!!!

**Review by Gimhan-minion — APPROVED** [review]
Thank you!
@AlexARNcode :100: :heart:

**Review by Piumal1999 — APPROVED** [review]

**Comment by Piumal1999** [comment]
Good Job @AlexARNcode
Thank you!!

**Pull request merged**
